### ISSUE TITLE
Missing first letter in file name

### ISSUE THREAD

**Issue by ikarus23** [patrickhener/goshs#33]
Hi Patrick!

I've noticed the first letter of files is missing after transfer.

Server: `goshs -d  /absolute/path/to/folder/`
Client: "Select All" --> "Download Selected" --> extract downloaded .ZIP file --> Files are missing the first letter (e.g. "foobar" --> "oobar")

**Comment by patrickhener** [comment]
Hi,

Can you please specify the goshs version you used and also the OS running goshs and the OS downloading files?

I had Problems with this versions ago and thought I did fix it.

Cheers,
Patrick

**Comment by ikarus23** [comment]
Both systems were Arch. I'm pretty sure I've downloaded the latest version form the release page (0.2.0).

**Comment by ikarus23** [comment]
Hmmm, I might be mistaken. Sorry, let me double check!

**Comment by ikarus23** [comment]
Ok, I did make a mistake and still was on 0.1.8. But now I downloaded 0.2.0 and the issue is still there.

**Comment by ikarus23** [comment]
I think it is because I did
```
goshs -d /absolute/path/to/folder/
```
instead of
```
goshs -d /absolute/path/to/folder
```
(`/` at the end)

**Comment by patrickhener** [comment]
Okay thanks for clarifying. Are there only files or do folder also miss the first Letter? And also do the files in those folder miss them too? Relevant might also be if you browse down a few folder before select All download or directly in the root itself.

I will look into the issue.

**Comment by patrickhener** [comment]
> I think it is because I did
> ```
> goshs -d /absolute/path/to/folder/
> ```
> instead of
> ```
> goshs -d /absolute/path/to/folder
> ```
> (`/` at the end)

That is interesting as well. I will try both Variants. This might be a quick fix then.

**Comment by patrickhener** [comment]
Hi,

i pushed a fixed version. If you need it right now just pull and install manually. Otherwise I will release v0.2.1 next time I implement a new feature or do bigger changes.

Thanks again for taking the time to report this issue and make `goshs` a little better this way. And also thanks for using it obviously :smile:

**Comment by patrickhener** [comment]
Is solved with merge request #36
